packages/data-stores/src/reader/types/index.ts

```diff
@@ -44,6 +44,13 @@ type SiteSubscriptionDeliveryMethods = {
 	};
 };
 
+export type PagedQueryResult< TDataType, TKey extends string > = {
```

Comment:
I suppose we could use `InfiniteData` from `react-query` but yours is a bit more precise to our case. 👍